test/pkg.jl

```diff
@@ -75,6 +80,10 @@ temp_pkg_dir() do
     @test isempty(Pkg.installed())
     @test !isempty(Pkg.available("Example"))
     @test !in("Example", keys(Pkg.installed()))
+    Pkg.rm("Example") #second rm to make sure things are ok
```

Comment:
could have changed the comment to something like "make sure calling Pkg.rm twice works" but whatever